images/win/scripts/Installers/Install-BizTalkBuildComponent.ps1

```diff
@@ -0,0 +1,89 @@
+################################################################################
+##  File:  Install-BizTalkBuildComponent.ps1
+##  Desc:  Install BizTalk Project Build Component
+################################################################################
+
+function Install-Msi
```

Comment:
We already have helper function to install MSI: https://github.com/actions/virtual-environments/blob/main/images/win/scripts/ImageHelpers/InstallHelpers.ps1#L1

Reply:
Explained in the other conversation.